README.md

```
@@ -1 +1,101 @@
+helm uninstall coolify
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the Coolify chart and their default values.
+
+| Parameter | Description | Default |
+|-----------|-------------|------|
+| `global.namespace` | Namespace to install Coolify | `coolify` |
+| `global.registryUrl` | Docker registry URL | `ghcr.io` |
+| `global.storageClassName` | Default storage class |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)